legal-api/src/legal_api/services/business_details_version.py

```diff
@@ -216,6 +216,19 @@ def get_business_revision(transaction_id, business) -> dict:
             .order_by(business_version.transaction_id).one_or_none()
         return VersionedBusinessDetailsService.business_revision_json(business_revision, business.json())
 
+    @staticmethod
+    def get_business_revision_before_filing(filing_id, business_id) -> dict:
+        """Consolidates the business info as of a particular transaction."""
```

Comment:
We have 3 methods now with the same description, could you make this reflect more on what is intended?

Reply:
Fixed.